test-case/check-kmod-load-unload.sh

```diff
@@ -97,15 +97,12 @@ do
         exit 1
     fi
 
-    dlogi "checking for fw_boot success"
-    keyword_info=$(dmesg |grep sof-audio | grep 'boot complete')
-    if [[ ! "$keyword_info" ]]; then
-        dloge "Error: Boot Complete not found in dmesg, fw_boot empty"
-        exit 1
-    fi
+    dlogi "checking if firmware is loaded successfully"
+    $(dirname ${BASH_SOURCE[0]})/verify-sof-firmware-load.sh > /dev/null
+    [[ $? -ne 0 ]] && dloge "Falied to load firmware after module insertion" && exit 1
```

Comment:
So we don't check the module load successfully or not?

Reply:
@libinyang  The problem is HERE: the kernel has made the log less verbose, we don't have the keyword "firmware boot complete" to check now, it's disappeared. see  https://github.com/thesofproject/linux/pull/1980. So, we should use a common log keyword both exist in verbose and less verbose log mode. Common keyword candadates:`Firmware debug build | Firmware info`.  They are both checked in `verify-sof-firmware-load.sh`, so, Instead of checking keywords, Use a subtest to do the same thing and ensure firmware is OK after module insertion.

Reply:
before firmware load check, we have grep any errors in dmesg, so a check on module load successfully seems not so necessary.

Reply:
@aiChaoSONG Sorry for my comments after merge. I was reviewing your patch and writing the comments and after I finished the review, I found it's already merged. : )
Yes, I understand that the keyword is removed because of the kernel change and this makes our old method not work. What I mean is even firmware is loaded successfully, kernel module load may still fail.

Reply:
@libinyang We may add the check later, the possible failure you mentioned is not checked before.